monai/utils/enums.py

```diff
@@ -311,3 +311,15 @@ class JITMetadataKeys(Enum):
     TIMESTAMP = "timestamp"
     VERSION = "version"
     DESCRIPTION = "description"
+
+
+class BoundingBoxMode:
```

Comment:
May I know why not define the class as a subclass of `Enum`?

Thanks.

Reply:
`BoundingBoxMode.XYXY` is not a string. You can find all the python Enum APIs in this page: https://docs.python.org/3/library/enum.html.

Thanks.